modules/social_features/social_event/social_event.module

```diff
@@ -307,3 +312,28 @@ function _social_event_format_date($event, $view_mode) {
 
   return $event_date;
 }
+
+/**
+ * Custom permission check, to see if people have access to users' events.
+ *
+ * Implements hook_block_access().
+ */
+function social_event_block_access(Block $block, $operation, AccountInterface $account) {
+
+  if ($operation === 'view' && ($block->getPluginId() === 'views_exposed_filter_block:events-events_overview' || $block->getPluginId() === 'views_block:events-block_events_on_profile')) {
+    // Here we're going to assume by default access is not granted.
+    $topicController = SocialEventController::create(\Drupal::getContainer());
```

Comment:
@jochemvn Minor thing, but i think this variable should be named `$eventController` ?